frontend/src/lib/search.ts

```diff
@@ -51,3 +52,21 @@ export function queryStatusCount(cluster: string): IRequestResult<ISearchResult>
             'query searchResult($input: [SearchInput]) {\n  searchResult: search(input: $input) {\n    count\n    related {\n      kind\n      count\n      __typename\n    }\n    __typename\n  }\n}\n',
     })
 }
+
+export function queryCCXReports(cluster: string): IRequestResult<ISearchResult> {
+    return postRequest<SearchQuery, ISearchResult>(backendUrl + apiSearchUrl, {
+        operationName: 'searchResult',
+        variables: {
+            input: [
+                {
+                    filters: [
+                        { property: 'kind', values: ['policyreport'] },
+                        { property: 'namespace', values: [cluster] },
```

Comment:
The policy report is a hub side resource?

Reply:
Yes as of now we are creating a PolicyReport CR on the hub for every violation per cluster.

Reply:
Cool, if that model stays the same, I think we should not use search then; it would be better to just make kube calls.